data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1843 @@
+			`	It all grows quiet for a while, and then the door opens again. Some of the other Heliarchs leave in a haste, and you're asked to head inside again. Consul Aulori is standing up still, giving you a nod as you come in.`
+			`	"Adjusted, your ship's sensors will be. Better suited, they will become, for detailed tridimensional projections," he says. He has a guard bring you back to your ship, where some engineers go inside your ship and spend some time tinkering with it. After they're done, one of the Heliarchs that was in the room approaches you, telling you they need you to land on the planet you described, <stopovers>, and to travel to <waypoints> to scan the ringworld. He wishes you luck, then hurries back to the restricted areas.`
+				accept
+	on stopover
+		dialog `You fly around the mysterious world for a while, until your sensors beep in an odd way, in what you guess is the signal that the Heliarch's modifications are done doing their read on the planet.`
+	on enter "World's End"
+		dialog `You fire up your ship's sensors, letting it collect the data the Heliarch want for hours on end. Eventually you hear a series of beeps in a dull rhythm, and look to see that their performance is back to normal.`
+	on complete
+		event "heliarch expedition break" 22 34
+		payment 3000000
+		conversation
+			`You're met by Aulori when you land, as well as several other Heliarchs. You transmit your scan logs, and they bring you to a separate room once again, where a model of the broken ringworld is being projected for all to see.`
+			`	"Skeptical of what you told us, we all were, Captain," Aulori says, gesturing for you to accompany him as he walks around the projection. "Claimed that insane, you were, some of those here did," he reaches down to tap a Kimek on her head, "Or that wasting our time with jokes, you were," he bumps another Saryd's arm, "Assured me, some of them had, that wholly impossible, it was, to break a ring," he pushes an Arach's leg with his hoof. He stops where the projection shows signs of greater damage, where a large chunk of the ring was cut off, littered with floating debris. "But not insane, neither a joker, you are. And impossible, this clearly is not. Invaluable information, what you've brought us is, Captain <last>. Investigate the site thoroughly, we must, so ask for your assistance again, we will, when a suitable crew for it, we have found."`
+			`	A Saryd rushes closer once Aulori says that, "Sir, against sending off anyone, the council is-".`
+			`	"Agree, the council will," Aulori cuts him off, still looking at the projection.`
+			`	The Saryd thinks for a while, then continues, "Take years, it would, to find and train someone willing to become an inspector. A dangerous task it-"`
+			`	"And wait around for years, we will not," Aulori interrupts him again. "Either find someone willing, or find another line of work, you will," he looks at the Saryd, then at the door.`
```

Comment:
I thought this was a 10 billion strong meritocracy. Where is this skill shortage from? Where is the preparation and planning?
Not to mention, where is this supremely blind arrogance inside their own ranks coming from? If the Heliarchy is half the meritocracy it claims to be then no one could have reached Aulori's position by being unable to listen to the well-informed advice of his relevant subordinates.

Reply:
Not meant to imply a skill shortage.
Leaving the Coalition is something that was prohibited by the Heliarchs a long time ago, and they've stopped sending out expedition parties of their own after the last all of them didn't make it back.

This isn't Aulori implying a skill shortage or ignoring feedback, it's Aulori implying that he doesn't care if this has been prohibited, he wants a thorough investigation on this ring, no matter what some other consuls might say.

I'll change up some bits of this chain to have it be more in line with him trying to keep this whole expedition section a secret, probably add another mission at the end too, after the Punisher has returned, where Aulori is in some discussion with other consuls about having done this in secret.